addOns/grpc/src/main/java/org/zaproxy/addon/grpc/internal/HttpPanelGrpcView.java

```diff
@@ -42,9 +43,11 @@ public class HttpPanelGrpcView implements HttpPanelView, HttpPanelViewModelListe
     private static final String CAPTION_NAME = Constant.messages.getString("grpc.panel.view.name");
 
     private HttpPanelGrpcArea httpPanelGrpcArea;
-    private JPanel mainPanel;
+    private static JPanel mainPanel;
```

Comment:
This should not be static, in which case the other method shouldn't be either.

Reply:
if i don't make mainPanel static i can't use it in , it suggesting mainPanel should be static
```
private static void showInvalidMessageFormatError(String message) {
        JOptionPane.showMessageDialog(
                mainPanel,
                message,
                Constant.messages.getString("grpc.encoder.message.invalid.format.error"),
                JOptionPane.ERROR_MESSAGE);
    }
```